content/blog/2024-09-10-memory-access-amortization.md

```diff
@@ -0,0 +1,158 @@
++++
+# `title` is how your post will be listed and what will appear at the top of the post
+title= "Memory Access Amortization in Valkey "
```

Comment:
I guess we might also want to briefly brain storm some better names here that better hook readers.

```
Maximizing the performance from your hardware with memory access amortization
```

Reply:
If you change it, you might also want to change the filename so the URL reflects the new title, but not totally required.

Reply:
I've not been making them consistent for the past few intentionally because I like short URLs but descriptive titles, but not a strongly held opinion.

Reply:
@madolson yeah, I'm with you on this one. As long as they aren't too far apart it's good to go shorter.